R/deduplicate.R

```diff
@@ -79,6 +79,13 @@ orderly_deduplicate_info <- function(config) {
                           fsep = "/")
 
   paths <- file.path(config$root, "archive", files$path)
+  exists <- vlapply(paths, file.exists)
+  if (any(!exists)) {
+    stop(paste0("Cannot deduplicate archive as database references files ",
```

Comment:
This is nicer to construct with `paste` and drop the trailing whitespace at the end of each string